### DISCUSSION TITLE
Windows 10 IoT Enterprise LTSC + AtlasOS?

### DISCUSSION THREAD

**Discussion by MariuzM** [Atlas-OS/Atlas#1169] [Help & Support] [answered]
Can i use both or its preferred to use regular Windows 10 Pro?

**Comment by Ast3risk-ops** [comment]
LTSC isn't supported and never will be.

**Reply by MariuzM** [reply]
Curious why is this the case?

**Reply by trolljoe** [reply]
> Curious why is this the case?
not only is it piracy to get the isos (msdl), it's hard to maintain 3 different versions

**Reply by Ast3risk-ops** [reply]
> Curious why is this the case?

- LTSC is expensive as fuck and only for organizations. Most people get it illegally.

- LTSC already has most windows components removed, doesn't require further optimization.

**Reply by pencilnav** [reply, marked as answer]
https://docs.atlasos.net/install-faq/windows-version-support/#which-windows-editions-are-supported
